event/src/unvalidated/event.rs

```diff
@@ -229,9 +248,11 @@ impl<D> From<Box<TimeEvent>> for Event<D> {
     }
 }
 
-impl<D> From<init::Payload<D>> for Event<D> {
-    fn from(value: init::Payload<D>) -> Self {
-        Self::Unsigned(value)
+// TODO : Delete this? then delete all
+// Delete event with new
```

Comment:
@nathanielc All cannot be delete since rest of the two impls are being used in migration code. Makes sense to keep all 3 then?

Reply:
Yes lets keep them